.github/workflows/test_and_deploy.yml

```diff
@@ -112,11 +168,17 @@ jobs:
     # alternatively, to publish when a GitHub Release is created, use the following rule:
     # if: github.event_name == 'release' && github.event.action == 'published'
     steps:
-      - uses: actions/download-artifact@v3
+      - uses: actions/download-artifact@v4
+        with:
+          # unpacks all the wheels into dist/
+          pattern: wheels-*
+          path: dist
+          merge-multiple: true
+          
+      - uses: actions/download-artifact@v4
```

Comment:
This bit is key. Because of using the v4 of the upload, need to use the v4 download. So this takes all those individual artifacts and merges them for the upload. This is also based on the scikit-image action.